### DISCUSSION TITLE
Device tracker e huawei

### DISCUSSION THREAD

**Discussion by sky7176** [indomus/forum#14] [HUB: Home Assistant]
Buon giorno a tutti,
dopo i casini degli aggiornamenti di google di ieri, devo aver combinato qualche casino e il mio P30 risulta sempre a casa.
Mi ricordo che nel vecchio forum avevo trovato le spiegazioni per sistemare il tutto, ma ora con il cambio di piattaforma non riesco più a trovare il post.

Vi ricordate il link in modo da poterci tornare?

GRazie
Giuseppe

**Comment by indomus** [comment]
https://spectrum.chat/indomus/home-assistant/huawei-p30-device-tracker~3e9ee8e9-e697-4dc7-9319-e87bd54b394a

questo forse?

**Reply by sky7176** [reply]
Si anche questa, ma mi ricordo che si parlava anche di autorizzare le varie app per usare il GPS, non avevo impostato il segnalibro sul brouser.

Per il momento il mio cellulare è ancora a casa anche se io sono a 40 km di distanza e ho il cellulare con me HI

**Comment by essegi65** [comment]
Io sto provando a risolvere lo stesso problema (con lo stesso telefono) da diversi mesi, credo che il problema sia dell'app di HA in quanto pur avendo le autorizzazioni del GPS in background sia il telefono che l'app stessa la posizione non viene aggiornata fino a quando non entro nell'app di HA. Una volta entrato all'interno dell'app la posizione cambia subito (quasi sembra che il sensore del GPS non si aggiorni e lo faccia solo con app aperta)

**Reply by snakuzzo** [reply]
Come già consigliato su Spectrum tempo fa, provate altre app come Life360...la cui integrazione è prevista nativamente su HA

**Comment by sky7176** [comment]
Ora si è messo a funzionare.
Essegi65 io ho autorizzato l'app mobile home per funzionare sempre anche in backgroud.
